### ISSUE TITLE
Failure during `docker-compose build` -- MongoDB install issue

### ISSUE THREAD

**Issue by bill-baumgartner** [JHUAPL/PINE#14]
Hi, I've encountered an issue when building via `docker-compose build`. Note that the log supplied by @bensonml  in Issue #10 shows success for the step where I am encountering a failure, so this may be specific to my environment somehow. I've not been able to get around it however, so I'm wondering if you have any suggestions.

System: `macOS Catalina 10.15.6`
Docker: `Docker version 19.03.13, build 4484c46d9d`; `docker-compose version 1.27.4, build 40524192`
Repo commit: `55695e9`

Steps taken:
1. `./pipelines/download_resources.sh`
2. set `VEGAS_CLIENT_SECRET` in `.env`
3. `export PINE_VERSION=$(./version.sh)`
4. Failed during `Step 19/28` when running `docker-compose build`

Output shown below:
```
Step 19/28 : RUN if [ -n "${DB_DIR}" ] ; then     apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4 &&     echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" > /etc/apt/sources.list.d/mongodb-org-4.0.list &&     apt-get -y update &&     apt-get install -y mongodb-org mongodb-org-server mongodb-org-tools mongodb-org-shell;     fi

[SKIPPING OUTPUT UNTIL ERROR]

Adding user mongodb to group mongodb
Done.
/var/lib/dpkg/info/mongodb-org-server.postinst: 43: /var/lib/dpkg/info/mongodb-org-server.postinst: systemctl: not found
dpkg: error processing package mongodb-org-server (--configure):
 installed mongodb-org-server package post-installation script subprocess returned error exit status 127
dpkg: dependency problems prevent configuration of mongodb-org:
 mongodb-org depends on mongodb-org-server; however:
  Package mongodb-org-server is not configured yet.

dpkg: error processing package mongodb-org (--configure):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin (2.27-3ubuntu1.2) ...
Errors were encountered while processing:
 mongodb-org-server
 mongodb-org
E: Sub-process /usr/bin/dpkg returned an error code (1)
Service 'eve' failed to build : The command '/bin/sh -c if [ -n "${DB_DIR}" ] ; then     apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 9DA31620334BD75D9DCB49F368818C72E52529D4 &&     echo "deb [ arch=amd64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 multiverse" > /etc/apt/sources.list.d/mongodb-org-4.0.list &&     apt-get -y update &&     apt-get install -y mongodb-org mongodb-org-server mongodb-org-tools mongodb-org-shell;     fi' returned a non-zero code: 100
```

Full log attached: [docker.log.txt](https://github.com/JHUAPL/PINE/files/5518861/docker.log.txt)

**Comment by bill-baumgartner** [comment]
Update: I've also tried installing in a Google Cloud VM but the build fails in the same place as described above.

**Comment by bwkchee** [comment]
Thank you for the detailed bug report.  We're currently exploring our options, it is due to an update in the mongo package.  Hope to have this resolved shortly.

**Comment by jhuapl-lglenden** [comment]
This should be resolved by #15 which updates the MongoDB package being used.

**Comment by bill-baumgartner** [comment]
Thanks for the fix. The build now succeeds.
